### PR TITLE
Extra includes and std::make_unique replacements

### PR DESCRIPTION
**Pull request by mattdawkins**

### CODE REVIEW

File: sealtk/core/test/KwiverVideoSource.cpp [hunk 1]
```diff
@@ -74,7 +74,9 @@ void TestKwiverVideoSource::init()
   videoReader->open(
     stdString(SEALTK_TEST_DATA_PATH("KwiverVideoSource/list.txt")));
 
-  this->videoSource = std::make_unique<core::KwiverVideoSource>();
+  this->videoSource =
+    std::unique_ptr<core::KwiverVideoSource>(new core::KwiverVideoSource());
```
Comment (mwoehlke-kitware) [inline sealtk/core/test/KwiverVideoSource.cpp]: Please prefer `{}`s to initialize.

File: sealtk/gui/Player.cpp [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include <QtGlobal>
 
 #include <memory>
+#include <utility>
```
Comment (mwoehlke-kitware) [inline sealtk/gui/Player.cpp]: BTW, what is the error without this?

File: sealtk/noaa/gui/Player.cpp [hunk 3]
```diff
@@ -41,7 +41,7 @@ Player::Player(QWidget* parent)
   QTE_D();
 
   d->loadDetectionsAction =
-    std::make_unique<QAction>("Load Detections...", this);
+    std::unique_ptr<QAction>(new QAction("Load Detections...", this));
```
Comment (mwoehlke-kitware) [inline sealtk/noaa/gui/Player.cpp]: Please hold off on this, #11 makes it go away.

File: sealtk/noaa/test/ImageListVideoSourceFactory.cpp [hunk 4]
```diff
@@ -21,6 +21,7 @@
 #include <QtTest>
 
 #include <memory>
+#include <utility>
```
Comment (mwoehlke-kitware) [inline sealtk/noaa/test/ImageListVideoSourceFactory.cpp]: Same question...

**Comment by mattdawkins** [comment]
Solved on other branch

**Pull request closed without merging**